src/scintillator/context.js

```diff
@@ -65,22 +65,21 @@ export class Context {
     let onTouch = e => {
       touches = [].slice.call(e.touches)
     }
-    let view = this.view
```

Comment:
To fix the test, can allow specifying an option, `touchEventTarget` in `Context` constructor. If not found, defaults to `this.view`.